code/ai/aibig.cpp

```diff
@@ -777,11 +777,23 @@ void ai_big_chase_attack(ai_info *aip, ship_info *sip, vec3d *enemy_pos, float d
 		
 		ai_turn_towards_vector(enemy_pos, Pl_objp, nullptr, rel_pos, 0.0f, 0, rvec);
 
+		// set an optimal range if applicable
+		float optimal_range = 0.0f;
+		ship_weapon* weapons = &Ships[Pl_objp->instance].weapons;
+		weapon_info* wip = nullptr;
+		if (weapons->num_primary_banks >= 1 && weapons->current_primary_bank >= 0) {
```

Comment:
We need a line in between `weapon_info* wip = nullptr;` and `if (weapons->num_primary_banks >= 1 && weapons->current_primary_bank >= 0) {` and at the bottom before the next if.  It helps when reading code to be able to identify blocks at a glance.